apps/ledger-live-desktop/tests/specs/speculos/nft.spec.ts

```diff
@@ -59,6 +56,10 @@ test.describe("send NFT to ENS address", () => {
       await app.sendDrawer.expectNftInfos(transaction);
     },
   );
+
+  test.afterAll(async () => {
+    delete process.env.DISABLE_TRANSACTION_BROADCAST;
```

Comment:
Would this affect following tests as we are removing the value ?
Should we restore the original value ?